### DISCUSSION TITLE
How to download sequences based on location information

### DISCUSSION THREAD

**Discussion by meiva101** [GMOD/jbrowse-components#4588] [Q&A]
How to download sequences based on location information

**Comment by cmdcolin** [comment]
Hi there, I didn't see this question when it first came in. One way is to use the "rubberband select" (click and drag on the scale bar) which will produce a menu that says "Get sequence"

![image](https://github.com/user-attachments/assets/2c6ee5a2-a0c8-4b28-8430-da18279b9cb1)


The other method is to click on a feature that covers your region of interest, and then use "Show feature sequence" in the feature details panel

We don't currently have a feature to input arbitrary coordinates via a text box but that could be a useful feature for us to add
